datastore/writer/postgresql_backend/sql_occ_locker_backend_service.py

```diff
@@ -130,8 +131,9 @@ def get_locked_collectionfields(
                         cf_lock,
                     )
                 )
-                filter_parts.append("(cf.collectionfield=%s and e.position>%s)")
+                filter_parts.append("(cf.collectionfield=%s and cf.position>%s)")
```

Comment:
This looks like a bug fixed. Can you write a regression test if so?

Reply:
It isn't; while looking through the code to write the docs I just noticed that the `position` column (which we specifically introduced for this lock) wasn't used in locking at all, so I reduced this case to a simpler query ;)

Reply:
So it can be deleted from the `schema.sql`?

Reply:
No, it is used now, see the line above. Previously it could have been removed (I think), but since it is updated and tested correctly, it would have generated a lot of code changes, so I think it would not have been worth the work.

Reply:
ok